src/config.toml

```diff
@@ -16,4 +16,10 @@ theme = "buildpacks"
   identifier = "docs"
   name       = "Docs"
   url        = "/docs"
-  weight     = 0
+  weight     = 2
+
+  [[menu.main]]
+  identifier = "blog"
+  name       = "Blog"
+  url        = "https://medium.com/"
```

Comment:
Do we need to wait to merge this until we have a proper URL, or can we merge it now?

Reply:
Hey @jkutner. Need to wait until we have a proper URL—have you created the publication in Medium?

Reply:
I updated this to point to the buildpacks medium i created.